src/network/networkd-link.c

```diff
@@ -916,6 +920,34 @@ int link_route_remove_handler(sd_netlink *rtnl, sd_netlink_message *m, void *use
         return 1;
 }
 
+static int link_enter_set_neighbors(Link *link) {
```

Comment:
the idea was always that functions called "…_enter_…" would enter a state. If there is no state associated with this, then they should have a different name. Maybe "request", "submit", "enqueue" or so would be a better word.

Reply:
Any of these sound good, but I went with request.